code/src/components/SubmitButton.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export const SubmitButton = () => {
+    return (
+        <div className='submit-button'>
```

Comment:
we could add a tab index on the submit button as it does not navigate there using just tab key at the moment.